### DISCUSSION TITLE
Element criteria in control panel filters - add "is not related to" to relation field based filters

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#15333] [Ideas] [answered]
Right now these have: "is related to", "has value" and "is empty". 
It would be useful if we could use "is not related to".

**Comment by brandonkelly** [comment, marked as answer]
`notRelatedTo` + `andNotRelatedTo` element query params, plus a corresponding `notRelatedTo` GraphQL argument and “Not Related To” element condition rule, are all coming in Craft 5.4. (#15496)

**Reply by brandonkelly** [reply]
Craft 4.12.0 and 5.4.0 are out with those changes.

**Comment by piotrpog** [comment]
@brandonkelly 
Thanks!
